pkg/controller/daemon/daemon_controller.go

```diff
@@ -803,16 +829,16 @@ func (dsc *DaemonSetsController) podsShouldBeOnNode(
 				inBackoff := dsc.failedPodsBackoff.IsInBackOffSinceUpdate(backoffKey, now)
 				if inBackoff {
 					delay := dsc.failedPodsBackoff.Get(backoffKey)
-					klog.V(4).Infof("Deleting failed pod %s/%s on node %s has been limited by backoff - %v remaining",
-						pod.Namespace, pod.Name, node.Name, delay)
+					logger.V(4).Info("Deleting failed pod on node has been limited by backoff",
+						"pod", klog.KRef(pod.Namespace, pod.Name), "node", klog.KObj(node), "delay", delay)
 					dsc.enqueueDaemonSetAfter(ds, delay)
 					continue
 				}
 
 				dsc.failedPodsBackoff.Next(backoffKey, now)
 
 				msg := fmt.Sprintf("Found failed daemon pod %s/%s on node %s, will try to kill it", pod.Namespace, pod.Name, node.Name)
-				klog.V(2).Infof(msg)
+				logger.V(2).Info(msg)
```

Comment:
Instead of logging the unstructured message, please convert to structured logging.

Reply:
As"msg" is needed below, keep that unchanged.

Reply:
This is still open.

Reply:
See previous comments.